tile/lib/lib.cc

```diff
@@ -493,6 +493,22 @@ lang::RunInfo LoadSoftmax(const std::string& name,      //
   return Evaluate(name, {E / N});
 }
 
+Tensor BatchNormalization(const Tensor& I, const Tensor& M, const Tensor& V, 
+                          const Tensor& G, const Tensor& B, const Tensor& E) {
+  return ((I - M) * G / sqrt(V + E)) + B;
+}
+
+lang::RunInfo LoadBatchNormalization(const std::string& name,      //
+                                     const LogicalShape& input) {  //
+  auto I = Placeholder(input);
+  auto M = Placeholder(input);
+  auto V = Placeholder(input);
+  auto G = Placeholder(input);
+  auto B = Placeholder(input);
```

Comment:
I would expect these to have the batch dimension changed to 1 rather than left the same as on the input. In the place you're using this, it already is size 1, but that is itself a bit weird. It would probably also be good to have broadcast tested and this would be a natural place to do it. So perhaps drop the first dimension from `input` to create the shape for these Tensors and see if things still work?